app/controllers/api/v1/admin/students/playlist_activities_controller.rb

```diff
@@ -0,0 +1,39 @@
+module Api
+  module V1
+    module Admin
+      module Students
+        class PlaylistActivitiesController < Api::ApiController
+          respond_to :json
+          before_action :set_student
+
+          def update
+            @student.playlist_activities.each do |activity|
+              activity.update(position: new_position(activity.id))
+            end
+            respond_with @student
+          end
+
+          private
+
+            # def authorize_resource
+            #   playlist_activity = Student.find(params[:student_id]).playlist_activities.first
```

Comment:
I think the appropriate authorization here is for the student record. `authorize(@student)` should suffice.